public/javascripts/floatingDots.js

```diff
@@ -6,12 +6,7 @@ function makeDots() {
             
             // SETTINGS 
             // opacity = 0.6,                                      // the opacity of the circles 0 to 1
-            colors = ['#ff9fcb',       // an array of rgb colors for the circles
-                      // 'rgba(189, 195, 199,' + opacity + ')',
-                      // 'rgba(241, 196, 15,' + opacity + ')',
-                      // 'rgba(231, 76, 60,' + opacity + ')',
-                      // 'rgba(231, 76, 60,' + opacity + ')'
-                     ],
+            colors = ['#ff9fcb'],
```

Comment:
Why is this file being modified in the leftPanel commit? I like to think of a commit like an essay - every line should be related.